zram-config

```diff
@@ -180,11 +180,14 @@ syncZdir() {
 serviceConfiguration() {
 	if [[ $1 == "stop" ]]; then
 		echo "Stopping services that interfere with zram device configuration." >> "$ZLOG"
-		if [[ $(systemctl is-active rsyslog.service) == "active" ]]; then
+		if ! [[ "$(grep -o '^ID=.*$' /etc/os-release | cut -d'=' -f2)" == "alpine" ]] && [[ $(systemctl is-active rsyslog.service) == "active" ]]; then
 			export rsyslogActive="true"
 			systemctl --no-block stop syslog.socket >> "$ZLOG" 2>&1 || return 1
+		elif [[ "$(rc-service syslog status | sed -ne 's/^ \* status: \(.*\)/\1/p')" == "started" ]]; then
```

Comment:
Out of curiosity, is this the only service that is writing to the `/var/log` directory by default on alpine? If not then you will likely run into issue with zram devices not resetting because of processes still attempting to write to the as a result of lazy unmounts.

Reply:
I don't think you need to use `sed` here. Can simply use exit codes and redirect output. For example, the condition statement below will return true only if syslog is started

````if rc-service syslog status 1> /dev/null 2> /dev/null; then````

Reply:
Even cleaner `if rc-service syslog status &> /dev/null; then`